src/firmware/mod.rs

```diff
@@ -114,6 +114,56 @@ pub enum Error {
     SecureDataInvalid,
 }
 
+impl std::fmt::Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let err_description = match self {
+            Error::IoError(_) => "I/O Error",
+            Error::InvalidPlatformState => "Invalid platform state",
+            Error::InvalidGuestState => "Invalid guest state",
+            Error::InvalidConfig => "Platform configuration invalid",
+            Error::InvalidLen => "Memory buffer to small",
+            Error::AlreadyOwned => "Platform is already owned",
+            Error::InvalidCertificate => "Invalid certificate",
+            Error::PolicyFailure => "Policy failure",
+            Error::Inactive => "Guest is inactive",
+            Error::InvalidAddress => "Provided address is invalid",
+            Error::BadSignature => "Provided signature is invalid",
+            Error::BadMeasurement => "Provided measurement is invalid",
+            Error::AsidOwned => "ASID is already owned",
+            Error::InvalidAsid => "ASID is invalid",
+            Error::WbinvdRequired => "WBINVD instruction required",
+            Error::DfFlushRequired => "`DF_FLUSH` invocation required",
```

Comment:
Why the backticks?

Reply:
There's an above description that commented in the `enum Error` that uses the backticks. I'll remove them.